zstacklib/zstacklib/utils/upload_task.py

```
@@ -0,0 +1,311 @@
+    def _fail_task(task, reason):
+        task.fail(reason)
+        raise Exception(reason)
+
+    def upload_slice(self, entity, param, task):
+        boundary = self.get_boundary(entity)
+        if not boundary:
+            self._fail_task(task, 'unexpected post form')
+
+        try:
+            self.stream_body(entity, boundary, param, task)
+        except Exception, e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)